.angular-cli.json

```diff
@@ -54,7 +54,7 @@
     }
   },
   "defaults": {
-    "styleExt": "css",
+    "styleExt": "scss",
```

Comment:
after changing this change all `.css` files to `.scss` files

Reply:
Already done in the last commit